README.md

```diff
@@ -278,11 +278,11 @@ require('telescope').setup{
         -- So, to not map "<C-n>", just put
         ["<c-x>"] = false,
         -- Otherwise, just set the mapping to the function that you want it to be.
-        ["<C-i>"] = actions.hselect,
+        ["<C-i>"] = actions.goto_file_selection_split,
```

Comment:
Example of what I mentioned in my comment. This isn't a change I'm making (as far as I can tell)... this is already what's on master.